charts/prometheus-adapter/templates/psp.yaml

```diff
@@ -4,9 +4,9 @@ apiVersion: policy/v1beta1
 kind: PodSecurityPolicy
 metadata:
   name: {{ template "k8s-prometheus-adapter.fullname" . }}
-  {{- if .Values.customAnnotations }}
+  {{- with (merge .Values.customAnnotations .Values.psp.annotations }}
```

Comment:
Have you tested this? It's missing a parenthesis

Reply:
Review comments were actioned via github.dev, so no - Tested properly now I'm free
<img width="438" alt="image" src="https://github.com/prometheus-community/helm-charts/assets/14006260/e1ef6366-4862-4082-9d2b-2d3df247bce0">